spec/ruby_llm/chat_tools_spec.rb

```diff
@@ -2,9 +2,49 @@
 
 require 'spec_helper'
 
+TEST_DELAY = 3.0
+
 RSpec.describe RubyLLM::Chat do
   include_context 'with configured RubyLLM'
 
+  # Helper to mitigate Bedrock rate limits in CI by retrying with backoff
```

Comment:
We only need this and the test delays when recording VCRs right? Would be nice to isolate it somehow. Bedrock is a bit of a pain isn't it?